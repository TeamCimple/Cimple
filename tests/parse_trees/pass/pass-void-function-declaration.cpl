void myFunc() {
	return;
}

int main() {
	return 0;
}
