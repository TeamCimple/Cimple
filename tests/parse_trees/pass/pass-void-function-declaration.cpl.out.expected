FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: RETURN(Int(0)))) , FuncDecl(Name: Var(myFunc) ReturnType: void Parameters:  Body: CompoundStatement(Declarations:  StatementList: RETURN())) 
